qiskit/pulse/transforms.py

```diff
@@ -12,19 +12,98 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Basic transformation functions which take a Schedule (and possibly some arguments) and return
-a new Schedule.
-"""
+"""Transformation passes and functions for pulse programs."""
```

Comment:
this should probably be in `passes` and be separated out rather than just "transforms"

Reply:
Agreed, just wanted to get this done quick to test performance on backends of optimized schedules.